### ISSUE TITLE
~/.modulesrc cannot contain modules 3 style module load calls

### ISSUE THREAD

**Issue by wenzler** [cea-hpc/modules#372]
### Describe the bug
Hello Xavier,
we found this problem with users using .modulesrc to get some initial modules preloaded.
I'm not sure whether this is even an intended use. 
The modules have a bit of tcl code in it - some if else, some exec ^^
I need to strip them of some info to paste later on.
Maybe you got an answer to this already.
I only tested 4.5.2 so far - can try with 4.6.1 next week.

Cheers

### To Reproduce

```console

cat << EOF > ~/.modulesrc
#%Module
module load git/2.15.1
module load gitlfs/2.5.2
EOF


reconnect


Module ERROR: can't read "::g_moduleEvalAttempt(/home/user/.modulerc)": no such variable

      while executing

  "module load git/2.15.1"

      (file "/home/user/.modulerc" line 2)

  Please contact <root@localhost>

Module ERROR: can't read "::g_moduleEvalAttempt(/home/user/.modulerc)": no such variable

      while executing

  "module load git/2.15.1"

      (file "/home/user/.modulerc" line 2)

  Please contact root@localhost
...
```

Location and content of any modulerc or modulefile involved:

```console
$ cat /home/user/.modulerc
...
```

### Expected behavior
Compatible behaviour as with modules v3

### Error and debugging information
See above

### Modules version and configuration
module version 4.5.2

### Additional context

Thanks!

**Comment by xdelaruelle** [comment]
Hello @wenzler 

User's `.modulerc` file is not the ideal place to load an initial set of modules (prefer shell rc file for that), but such error should not happen anyway.

I would need more details from your side as I am not able to reproduce currently. Could you put the following line at the beginning of Modules script in profile.d directory (usually `/etc/profile.d/modules.sh`)

    export MODULES_VERBOSITY=debug

Then provide the debug logs that are printed when reconnecting.

I am also interested to get the content of your `siteconfig.tcl`, `initrc` and `rc` files if any in etc directory (usually in `/etc/environment-modules`)

**Comment by xdelaruelle** [comment]
I can reproduce this issue on my side. The `auto_handling` configuration option should be disabled to trigger this issue. I am working on a fix that will be part of upcoming v4.7 release.
